src/CompDevSDKMain.c

```diff
@@ -0,0 +1,19 @@
+#include "CompDevSDK.h"
+
+#include "ports.h"
+
+TaskHandle_t BlinkTaskHandle;
+void BlinkTask(void *param);
+
+void CompDevSDKMain()
+{
+    setPinMode(13, PIN_OUTPUT);
```

Comment:
Wszystkie konfiguracje sprzetu dobrze jest wrzucic w zbiorcza funkcje system_init(). Taka notka na przyszlsoc.

Reply:
Ok, to od razu poprawię.